jose-emi/app.js

```diff
@@ -0,0 +1,316 @@
+var express = require('express');
+var app = express();
+var bodyParser = require('body-parser');
+
+app.use(bodyParser.json());
+
+const DIAS_MILI_SEG = 1000 * 60 * 60 * 24
+
+function Libro(titulo,id,cantidad){
+    this.id = id;
+    this.titulo = titulo; 
+    this.cantidad = cantidad;
+}
+
+function Socio(nombre,id){
+    this.id = id;
+    this.nombre = nombre;
+}
+
+function Prestamo(id,idLibro,idSocio, fechaVto){
+    this.id = id;
+    this.idLibro = idLibro;
+    this.idSocio = idSocio;
+    this.fechaVto = fechaVto;
+    
+}
+
+var socios = new Array();
+var libros = new Array();
+var prestamos = new Array();
+
+// REQ1.1 Añadir libros
+app.post('/libros',function(req,res){
+    if(req.body.titulo!=null){
+        if(req.body.cantidad>0 && Number.isInteger(req.body.cantidad)==true){
```

Comment:
What happens if the req.body.cantidad is not an integer, doesn't it brake?
Also, there is no need for == true
`if(Number.isInteger(req.body.cantidad) && req.body.cantidad>0 ){`